### DISCUSSION TITLE
The volume up and down bars are not displayed.

### DISCUSSION THREAD

**Discussion by FiRSTTX** [valinet/ExplorerPatcher#2017] [Q&A] [answered]
The volume up and down bars are not displayed. 
How to Fix ? or Not support this version ?

**Comment by pyrates999** [comment, marked as answer]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1692

Please close this one.

Work around until this gets fixed: install an application from the microsoft store called "modernflyouts".

**Reply by FiRSTTX** [reply]
thank you .
